### ISSUE TITLE
Fuzzing: Relay Connection Argument Exceptions

### ISSUE THREAD

**Issue by stephencelis** [rmosolgo/graphql-ruby#590]
Giving junk arguments for Relay pagination results in uncaught exceptions. _E.g._,

``` graphql
me {
  followers(first: -1) { # or `last: -1`
    nodes {
      id
    }
  }
}
```

This line is the culprit:

https://github.com/rmosolgo/graphql-ruby/blob/a15142a0c176df7ef9da9dc216eba87db7428527/lib/graphql/relay/array_connection.rb#L26

Not a big deal, but we probably don't want to completely blow up on bad client requests.

**Comment by rmosolgo** [comment]
Yes, I agree that client errors shouldn't result in Ruby exception! Thanks for reporting this one.

**Comment by rmosolgo** [comment]
This'll be fixed in 1.5.2!

**Comment by stephencelis** [comment]
🙌
